src/vm/exceptionhandling.cpp

```diff
@@ -4488,9 +4488,12 @@ VOID DECLSPEC_NORETURN UnwindManagedExceptionPass1(PAL_SEHException& ex, CONTEXT
 
     do
     {
+        CONTEXT currentContext = *frameContext;
```

Comment:
We should not change anything in this function, but rather fix it for x86 elsewhere. This function works for amd64 and arm, so it should be possible to ensure that it works for x86 too.
One of the problems here is that you set the dispatcherContext.ContextRecord to a local variable address, but the ProcessCLRException stores it in the ExceptionTracker that outlives this function. 
This is the reason why 65 tests have failed in the CI.